mubench.reviewsite/tests/TagsTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+require_once "DatabaseTestCase.php";
+
+use MuBench\ReviewSite\Controller\TagController;
+
+class TagTest extends DatabaseTestCase
+{
+
+    private $tagController;
+
+    function setUp()
+    {
+        parent::setUp();
+        $this->tagController = new TagController($this->db, $this->logger);
+        $this->db->table('tags')->insert(['name' => 'test1']);
```

Comment:
Please move the insertion into the test cases (where it is needed). There's actually only one test that needs all three tags. The others need at most one or even none. Use `setUp` to reduce duplication in tests, not to introduce additional duplication.